src/serac/physics/solid_mechanics.hpp

```diff
@@ -263,6 +263,7 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
     initializeSolidMechanicsStates();
   }
 
+#if 0
```

Comment:
?

Reply:
this is related to the disabled lua input decks